source/CGameVersionManager.cpp

```diff
@@ -28,6 +28,7 @@ namespace CLEO
         { 0x00465AA0,	memory_und, 0x00465AA0, 0x00465B20, 0x0046B2C0 },		// MA_STOP_SCRIPT_FUNCTION,
         { 0x00465E60,	memory_und, 0x00465E60, 0x00465EE0, 0x0046B640 },		// MA_SCRIPT_OPCODE_HANDLER0_FUNCTION,
         { 0x00464080,	memory_und, 0x00464080, 0x00464100, 0x00469790 },		// MA_GET_SCRIPT_PARAMS_FUNCTION,
+        { 0x00464250,	memory_und, memory_und, memory_und, memory_und },		// MA_GET_NEXT_SCRIPT_PARAM_NO_UPDATE_FUNCTION,
```

Comment:
So it is time to add check for US 1.0 version and throw error message for others.